baselines/ts5.5/audioworklet.generated.d.ts

```diff
@@ -347,7 +347,9 @@ declare var ByteLengthQueuingStrategy: {
  * [MDN Reference](https://developer.mozilla.org/docs/Web/API/CompressionStream)
  */
 interface CompressionStream extends GenericTransformStream {
+    /** The **`readable`** read-only property of the CompressionStream interface returns a ReadableStream. */
```

Comment:
Wow, what useless comments... Who is providing these?

Reply:
It's a shame we are wasting parsing time and memory on these comments that just restate the annotation 😞